streaming/src/main/scala/org/apache/spark/streaming/Checkpoint.scala

```diff
@@ -41,6 +41,13 @@ class Checkpoint(ssc: StreamingContext, val checkpointTime: Time)
   val checkpointDuration = ssc.checkpointDuration
   val pendingTimes = ssc.scheduler.getPendingTimes().toArray
   val sparkConfPairs = ssc.conf.getAll
+  @transient val recoverableAccuNameToId = ssc.recoverableAccuNameToId
```

Comment:
Hmmm, I don't think so, `ssc.` is used everywhere in this class. can you try it? probably the exception you met was lead by something else.